ouroboros-consensus/README.md

```diff
@@ -12,6 +12,10 @@ This package contains:
 * `docs`: documentation, in particular, `docs/report` contains the technical
   report about the consensus and storage layer.
 
+* `src-docs`: a collection of tutorials that are helpful for understanding the
```

Comment:
Why not put the new documentation inside `docs` to avoid the proliferation of directories at the `ouroboros-consensus` level and potential confusion?